GitUI/CommandsDialogs/BrowseDialog/DashboardControl/Dashboard.cs

```diff
@@ -18,7 +18,6 @@ public partial class Dashboard : GitModuleControl
     {
         private readonly TranslationString _cloneFork = new TranslationString("Clone {0} repository");
         private readonly TranslationString _cloneRepository = new TranslationString("Clone repository");
-        private readonly TranslationString _cloneSvnRepository = new TranslationString("Clone SVN repository");
```

Comment:
Is there an entry in a translation file that needs to be removed as well? I haven't looked much at the translation system yet.